apps/condo/domains/common/hooks/useNotificationMessages.tsx

```diff
@@ -0,0 +1,18 @@
+import { Typography } from 'antd'
+import React, { useCallback } from 'react'
+
+import { useIntl } from '@condo/next/intl'
+
+
+export const useNotificationMessages = () => {
+    const intl = useIntl()
+    const ChangesSavedMessage = intl.formatMessage({ id: 'ChangesSaved' })
+    const ReadyMessage = intl.formatMessage({ id: 'Ready' })
+
+    const getSuccessfulChangeNotification = useCallback(() => ({
+        message: <Typography.Text strong>{ReadyMessage}</Typography.Text>,
+        description: <Typography.Text type='secondary'>{ChangesSavedMessage}</Typography.Text>,
+    }), [ChangesSavedMessage, ReadyMessage])
```

Comment:
It’s also depends on intl

Reply:
It’s depends on translations. `intl` not using in `getSuccessfulChangeNotification`